bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -431,7 +431,7 @@ spec:
                       fieldPath: metadata.name
                 - name: OPERATOR_NAME
                   value: windows-machine-config-operator
-                image: REPLACE_IMAGE
```

Comment:
this change will break

https://github.com/openshift/windows-machine-config-operator/blob/8a4f95f905527ece51fa9fb2580764a7ff6645a4/hack/common.sh#L324

is there a workaround, or this hack feature will no longer work/supported?